packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -290,8 +290,6 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
    * Never undefined.
    */
   public readonly engine?: IClusterEngine;
-  public readonly instanceIdentifiers: string[] = [];
-  public readonly instanceEndpoints: Endpoint[] = [];
```

Comment:
Why move these? We keep members in a specific order (public, then protected, then private). Let's keep it this way here.

Reply:
They are actually moved to the implementing classes, so they no longer exist on the `DatabaseClusterNew` class. (diff looks almost as if they were just moved a bit down inside same class, but there's actually 200 rows in between)

The reason for moving them was that I could not figure out a way to keep them readonly on the base class if we needed to set them from the implementing classes.